contributing-snippets.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to Tiltfile Snippets
+Would you like to help improve our Snippet Library?  
+Submitting a snippet is as straightfoward as submitting a Pull Request.
+
+0. Fork this repository on Github and then clone from your forked version of this repository
+
+1. Create a file in src/_data/snippets/*snippet_name*.yml
+
+2. Copy the following content and modify as applicable
+```
+title: Descriptive Name of Your Snippet
+description: Explain what your snippet does
+code: |
+  *Your code goes here*
+contributor: Your Github Handle (optional)
+```
+
+3. Commit your changes and submit a Pull Request
+
+4. A member of the Tilt team will review your snippet and modify it if necessary.  
```

Comment:
What about this edit? To me, this seems more respectful that people are taking time out of their busy schedules to even submit a snippet, and expresses more appreciation and respect for that time.


> A member of the Tilt team will review your snippet and suggest edits if necessary. If there's a reason that we don't think it's right for our Snippets Library, we'll let you know why. We appreciate you taking the time to contribute!